ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+## Preamble
+
+```
+SEP: 0036
+Title: Regulated Assets Transacting
```

Comment:
I think this is a case where we are building a solution or one use case, but it may be relevant to others and we do not need to bind the definitions to terms like regulated assets. We are building a thin/small interface between wallets to get issuer approval.
```suggestion
Title: Transaction Approval for Assets Requiring Issuer Approval
```

Reply:
Because of what we say in `Simple Summary`, I feel it makes more sense to include `Regulated Assets` in the title. Hence, I came up with something that is the mix of your suggestions - `Transaction Approval for Regulated Assets` in https://github.com/stellar/stellar-protocol/pull/750/commits/8ec2d6d6dc55d5996a838ce4b2bbca1874f1c8d9